network-api/networkapi/wagtailpages/factory/homepage_features.py

```diff
@@ -44,6 +48,21 @@ def generate(seed):
 
     reseed(seed)
 
+    HomepageFocusAreas.objects.create(
+        page=home_page,
+        area=FocusArea.objects.get(name='Empower Action')
+    )
+
+    HomepageFocusAreas.objects.create(
+        page=home_page,
+        area=FocusArea.objects.get(name='Connect Leaders')
+    )
+
+    HomepageFocusAreas.objects.create(
```

Comment:
Could this be thrown into a for loop?

```python
for name in ['Empower Action', 'Connect Leaders', 'Investigate & Research']:
    HomepageFocusAreas.objects.create(
        page=home_page, 
        area=FocusArea.objects.get(name=name),
    )
```

Reply:
I considered it, but it doesn't really make things more efficient, and it doesn't really make things more readable either, so I left it as plain code. especially since this is factory code rather than "live API servicing" or the like.

Reply:
if you want to rewrite it, that's fine by me though.

Reply:
Nah, sounds like you thought about it already. We'll leave it as is. Resolving this now, thanks Pomax!